m2e-maven-runtime/org.eclipse.m2e.maven.runtime/pom.xml

```diff
@@ -26,7 +26,7 @@
 
 	<properties>
 		<!-- maven core version -->
-		<maven-core.version>3.8.3</maven-core.version>
+		<maven-core.version>3.8.1</maven-core.version>
```

Comment:
Below `<maven-resolver.version>1.6.3</maven-resolver.version>` should also be reverted to 1.6.2 in case we really revert in the end. But I don't know if this affects this test.